jlm/hls/ir/hls.hpp

```diff
@@ -80,6 +80,11 @@ class fork_op final : public jlm::rvsdg::simple_op
       : jlm::rvsdg::simple_op({ type }, std::vector<jlm::rvsdg::port>(nalternatives, type))
   {}
 
+  fork_op(size_t nalternatives, const rvsdg::type & type, bool constant)
+      : rvsdg::simple_op({ type }, std::vector<rvsdg::port>(nalternatives, type)),
+        Constant_(constant)
+  {}
+
   bool
   operator==(const jlm::rvsdg::operation & other) const noexcept override
```

Comment:
You added an attribute to the operation, but did not modify the equality operator. Is the equality operator still correct?